apps/src/gamelab/GameLab.js

```diff
@@ -16,7 +16,8 @@ import {
 } from '@cdo/apps/lib/util/javascriptMode';
 import JavaScriptModeErrorHandler from '@cdo/apps/JavaScriptModeErrorHandler';
 import BlocklyModeErrorHandler from '@cdo/apps/BlocklyModeErrorHandler';
-var msg = require('@cdo/gamelab/locale');
+var gamelabMsg = require('@cdo/gamelab/locale');
+var spritelabMsg = require('@cdo/spritelab/locale');
```

Comment:
Only one of these will ever be non-empty on a given page, yeah?

Reply:
Yes, my understanding is that is guaranteed to be true if apps can only access their own locale files (and seems true from local testing)

Reply:
Can we use that to detect which to load then?

```js
var msg = require('@cdo/gamelab/locale') || require('@cdo/spritelab/locale');
```

(Prob not worth doing... just verifying my understanding.)